sample_config.toml

```diff
@@ -0,0 +1,74 @@
+# This file contains some examples of Drone configuration.
+# You can run the drone with --dump-config to validate a configuration.
```

Comment:
this is not implemented yet?

Reply:
It's here: https://github.com/drifting-in-space/spawner/blob/a3dc709adb38451dfb73fb9a8c419aee868fefea/drone/src/drone/mod.rs#L50

Reply:
ah my bad i keep forgetting the - to _ thing while grepping